src/app/data/heroData.ts

```diff
@@ -0,0 +1,72 @@
+const heroClassesData: Array<{
```

Comment:
You can create an interface for these types, e.g. `Hero`, `Armor`, `Weapon`, etc. Not strictly needed, but would be a nice touch.